scalableminds/webknossos
# Add REST API versioning support

The API is subject to frequent changes. However, older versions will be supported for some time via prefixed routes.
 - To access the latest version, call the routes with only the `/api` prefix.
 - To access a specific version, call the routes with a prefix in the form of `/api/v1`.

Related: documentation in #3376 

### URL of deployed dev instance (used for testing):
- https://restapiversioning.webknossos.xyz

### Steps to test:
- try api calls with `/api/v1` prefix, should all work in the same way
- to test actual new version, uncomment the example lines in webknossosApi.routes and add a method to the LegacyApiController

### Issues:
- contributes to #3372

------
- ~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~ (let’s do that when we release the API docs)
